scalding-commons/src/main/scala/com/twitter/scalding/commons/scheme/CombinedSequenceFileScheme.scala

```diff
@@ -1,20 +0,0 @@
-package com.twitter.scalding.commons.scheme
```

Comment:
was deleted on cascading3 branch from scalding PR. Have not looked into it much.